index.html

```diff
@@ -332,6 +332,33 @@ <h3 class="lets-connect">LET’S CONNECT</h3>
       </div>
 
     </section>
+    <section class="contact">
+      <div class="contact-wrap">
+        <div class="contact-intro">
+          <h1 class="contact-title">Contact me</h1>
+          <p class="contact-text">
+            If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I’d love to help with it
+          </p>
+        </div>
+
+        <!-- contact form -->
+        <div class="form-wraper">
+          <form action="https://formspree.io/f/xrgjwwkj" class="contact-form" method="post">
+            <label for="name"></label>
+            <input class="form-control" type="text" name="name" id="name" maxlength="30" placeholder="Enter your name"
+              required>
+            <label for="email"></label>
+            <input class="form-control" type="email" name="email" id="email" placeholder="Enter your email" required>
+            <label for="message"></label>
+            <textarea class="form-control" name="message" id="message" placeholder="Write your message here"
+              maxlength="500" required></textarea>
+            <button type="submit" class="get-button" id="submit">Get in Touch</button>
+          </form>
+        </div>
+      </div>
+    </section>
```

Comment:
### Figma Template
![figma (copy)](https://user-images.githubusercontent.com/93039359/179044095-1e1a88a8-03d5-48c1-bb42-0a6708ea089f.png)

- [ ] Your design is almost similar to the Figma template :+1: but you forgot to add this bar shown in the Figma template. :fearful: Please add it to your design :+1: